src/constants/classes.ts

```diff
@@ -0,0 +1,22 @@
+export const classes = [
+    {
+        id: '0',
+        name: '1',
+    },
+    {
+        id: '1',
```

Comment:
why are these classes used ?

Reply:
I have removed the ```classes``` entity now. You can check now.